bigquery_etl/cli/query.py

```diff
@@ -1778,7 +1811,10 @@ def deploy(
             dataset_name = query_file_path.parent.parent.name
             project_name = query_file_path.parent.parent.parent.name
 
-            full_table_id = f"{project_name}.{dataset_name}.{table_name}"
+            if destination_table:
+                full_table_id = destination_table
```

Comment:
Looks like there's an assumption made here about what the user should enter in this param, `{project_name}.{dataset_name}.{table_name}` if this indeed is the case then perhaps we should validate the input matches this expectation.

Reply:
I'd take note to address this in a future iteration.